### PR TITLE
[Keyboard] Add KL-90

### PR DESCRIPTION
**Pull request by swiftrax**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/kikoslab/kl90/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "KL-90", 
+    "url": "www.kikoslab.com", 
+    "maintainer": "swiftrax", 
+    "width": 18, 
+    "height": 6.75, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (zvecr) [inline keyboards/kikoslab/kl90/info.json:8]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/kikoslab/kl90/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes
+OLED_DRIVER_ENABLE = yes
+WPM_ENABLE = yes
```
Comment (zvecr) [inline keyboards/kikoslab/kl90/rules.mk:25]: Shouldnt be enabled at the keyboard level, if required move to keymap
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Looks like the info.json doesn't match the number of keys in the layout_all macro.

**Comment by swiftrax** [comment]
> Looks like the info.json doesn't match the number of keys in the layout_all macro.

I believe I fixed it now

**Review by drashna — APPROVED** [review]

**Pull request merged**